website/frontend/src/components/footer/Footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import "./footer.css"
+
+const Footer = () => {
+    return (
+        <div className='footer justify--content'>
+            <span>Copyright © 2021</span>
```

Comment:
Should be `2023` instead of `2021`:
```suggestion
            <span>Copyright © 2023</span>
```